napari/__main__.py

```diff
@@ -14,9 +14,9 @@
 from textwrap import wrap
 from typing import Any, Dict, List
 
-from . import __version__, gui_qt, layers, view_path
-from .components.viewer_model import valid_add_kwargs
-from .utils import citation_text, sys_info
+import napari
+from napari.components.viewer_model import valid_add_kwargs
+from napari.utils import citation_text, sys_info
```

Comment:
@tlambert03 why are these absolute imports now? I don't think this is the correct approach. If I have an installed version of napari different from the one that runs this code, and I try `python __main__.py` from this working directory, it will run the installed version, rather than the version corresponding to this package.

Looking at the code below, I don't think that anything fundamentally requires these to become absolute imports?

Reply:
reverted